app/app.go

```diff
@@ -129,7 +129,7 @@ import (
 
 const (
 	Name       = "centauri"
-	dirName    = "banksy"
+	dirName    = "centauri"
```

Comment:
I don't think we can change the dirName after launch,

Reply:
why dp ypu think so?

there is data dir parameter

https://github.com/ComposableFi/composable/blob/30d3e09dcedf20755b4d29a9dbc5d32a18f9319a/inputs/notional-labs/composable-centauri/flake-module.nix#L513

or 

people 

can just run mv command to move dir

works both ways

Reply:
because this folder also has priv key. If we just copy it, there's a risk of double sign for validators.

Reply:
what ypu mean copy? same node dame person just points via data dir cli parameter to oldel folder or moves old folder to new folder and not parameters needed. if new folder creared it will.be nrw key not of validator.

so i do not see how what you describin might happe